File: withFunctions/randomNumbers.java
package springIntro;

import java.util.*;

public class randomNumbers {

	static void rand() {
		Random random = new Random();
		int number = random.nextInt(70) + 7;
		System.out.printf("Square of the %d = %d\n", number, frame(number));
	}
	
	static int frame(int number) {
		return number * number;
	}

	public static void main(String[] args) {
		for(int i = 0; i < 7; i++) {
			rand();
		}

	}

}

File: withFunctions/randomNumbers.cpp
#include<iostream>
#include<cstdlib>
#include<ctime>
using namespace std;
int frame(int number){
	return number * number;
}
void random(){
	int number = (int) rand() % 70 + 7;
	cout << "Square of the " << number << " = " << frame(number) << endl;
}
int main(){
	srand(time(NULL));
	for(int i = 0; i < 7; i++){
		random();
	}
	return 0;
}
